JSDemos/Demos/TabPanel/Overview/description.md

```diff
@@ -1,5 +1,13 @@
-<<<<<<< HEAD
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView UI components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
-=======
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
->>>>>>> e74b8529 (Cut off "UI" from "UI component" is certain cases (#612))
+The [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) UI component consists of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/Documentation/ApiReference/UI_Components/dxMultiView/) components. TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa. This demo demonstrates the following TabPanel features:      
```

Comment:
"TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa." - is it the same as "With TabPanel, users can press tabs to switch between views"?

Reply:
"This demo demonstrates the following TabPanel **features**: ", but the headers below are not feature nouns. What if use such a structure:
"""
To configure TabPanel, do the following: 
1) Bind the TabPanel to Data
2) Customize Tab Switch
"""
?

Reply:
>"TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa." - is it the same as "With TabPanel, users can press tabs to switch between views"?

It is not completely the same. Yes, users can press tabs to switch between views but the can also swipe. Moreover, it is not the main point. TabPanel consists of dxTabs and dxMultiView, and one of the core TabPanel features is to synchronize a tab and a view's content.

>To configure TabPanel, do the following: 
1) Bind the TabPanel to Data
2) Customize Tab Switch

When we start a paragraph saying "To do X, do the following", we mean some purpose. To configure a component is not a purpose but something we should do to reach it. I would suggest something like 'This demo shows the following TabPanel configuration:'

>Customize Tab Switch

The word configure fits better since we do not customize (to make it convenient for somebody) but specify some properties.